### PR TITLE
Iot 463 get devce

### PR DESCRIPTION
**Pull request by prashantdatirIngress**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/nodejs-iot/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

**Comment by prashantdatirIngress** [comment]
> @prashantdatirIngress Please add respective testing method in quickstarter.js, as that will help us to showcase end user when they will try to use ClearBlade SDK. (We can also use that directly to put on read.me file), you will find those examples in my last changes.

Updated

File: src/v1/device_manager_client.ts [hunk 1]
```diff
@@ -951,23 +952,41 @@ export class DeviceManagerClient {
...
+          const device: protos.google.cloud.iot.v1.IDevice = JSON.parse(data);
+          resolve([device, {}, {}]);
+        })
+      })
+      req.on('error', (e) => {
+        reject(e);
       });
-    this.initialize();
-    return this.innerApiCalls.getDevice(request, options, callback);
+      // if (payload) {
```
Comment (ronak-ingress) [inline src/v1/device_manager_client.ts:985]: Remove this if payload is not needed.

File: src/v1/device_manager_client.ts [hunk 2]
```diff
@@ -951,23 +952,41 @@ export class DeviceManagerClient {
...
-    options.otherArgs.headers['x-goog-request-params'] =
-      this._gaxModule.routingHeader.fromParams({
-        name: request.name || '',
+
+    return new Promise(async (resolve, reject) => {
+      
+
+      var options = {
+        host: 'iot-sandbox.clearblade.com',
+        path: `/api/v/4/webhook/execute/`+ adminSystemKey +`/devices?name=`+request?.name,
```
Comment (clarkbynum) [inline src/v1/device_manager_client.ts:961]: This should use a system key from a template system, not the admin system key. Also, the webhook is now called `cloudiot_devices`, not `devices`

**Review by clarkbynum — CHANGES_REQUESTED** [review]

File: src/v1/device_manager_client.ts [hunk 3]
```diff
@@ -951,23 +952,41 @@ export class DeviceManagerClient {
...
+        method: 'GET',
+        headers: {
+          'Content-Type': 'application/json',
+          'ClearBlade-UserToken': adminSystemUserToken          
+        }
+      };
+      
+      //console.log(options);
```
[the remark this answers is not in the document]
Comment (prashantdatirIngress) [reply src/v1/device_manager_client.ts]: Done

File: src/v1/device_manager_client.ts (hunk 2 above)
Comment (prashantdatirIngress) [reply src/v1/device_manager_client.ts]: Updated

**Pull request merged**